Monika After Story/game/script-topics.rpy

```diff
@@ -16165,3 +16165,38 @@ label monika_giving_criticism:
     m 3dsu "Graciously accept you can't change everyone's mind and stay considerate when assessing someone else's work."
     m 3hub "...That'd be Monika's Critique Tip of the Day, ahaha!"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_curse_words",
+            category=["advice", "life"],
+            prompt="Curse words",
+            random=False,
+            pool=False
+        )
+    )
+
+label monika_curse_words:
+    m 3etd "Say, {w=0.2}[player], {w=0.2}do you use curse words sometimes?"
+    m 1hksdlb "Ahaha, {w=0,2}I mean, {w=0,2}it's alright if you swear occasionally, {w=0.2}and I don't have anything against it if you really do."
+    m 5rtd "I even heard of some studies stating that using... {w=0.5}a bit stronger language helps easing the pain if you, say, hurt yourself with something by the accident."
+    m 5eta "Or that it really helps dumping your steam if you feel so angry about something."
+    m 7eud "But don't get me wrong, {nw}{w=0.2}"
+    extend 2dsd "I wouldn't like if you were spitting them every now and then, {nw}{w=0.2}"
```

Comment:
I'm unsure if `every now and then` is correct here. To me this more implies she's saying `I wouldn't like it if you were saying them at all`, since `every now and then` is essentially "occasionally". Which imo she'd be okay with.

Reply:
No, no, that's now what I meant. I mean, I'm friends with people who just spit these words right after every single normal one (and there are a lot of other people like that, that's for sure), and that's what I tried to express here.

Reply:
I've rephrased this line, so I'm going to resolve this conversation.